static/js/containers/LearnerSearchPage_test.js

```diff
@@ -0,0 +1,55 @@
+/* global SETTINGS: false */
+// import TestUtils from 'react-addons-test-utils';
+import { assert } from 'chai';
+import _ from 'lodash';
+
+import IntegrationTestHelper from '../util/integration_test_helper';
+import { ELASTICSEARCH_RESPONSE } from '../constants';
+
+describe('LearnerSearchPage', function () {
+  let listenForActions, renderComponent, helper, server;
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper();
+    listenForActions = helper.listenForActions.bind(helper);
+    renderComponent = helper.renderComponent.bind(helper);
+    server = helper.sandbox.useFakeServer();
+    server.respondWith("POST", "http://localhost:9200/_search", [
+      200, { "Content-Type": "application/json" }, JSON.stringify(ELASTICSEARCH_RESPONSE)
+    ]);
+  });
+
+  afterEach(() => {
+    helper.cleanup();
+  });
+
+  it('calls the elasticsearch API', () => {
+    assert(_.isEmpty(server.requests));
+    return renderComponent('/learners').then(() => {
+      assert(!_.isEmpty(server.requests));
+      let request = server.requests[0];
+      assert.deepEqual(request.url, "/_search");
+      assert.deepEqual(request.method, "POST");
+    });
+  });
+
+  it('displays search results', function (done) {
+    renderComponent('/learners').then(([, div]) => {
+      this.timeout(10000);
+      setTimeout(() => {
+        let results = document.querySelectorAll('.learner-result');
+        console.log(results);
+        console.log(div);
+        done();
+      }, 1000);
```

Comment:
this is WIP code, I was having a lot of issues getting the integration tests working. I'll try to put more time into it today.